copying_headers/test-license

```diff
@@ -210,7 +210,9 @@ class Repo(object):
 
         # Create a list of paths to partial manifest files
         self.manifest_list = [os.path.join(self.copying, path)
-                              for path in listing if path.startswith('MANIFEST.')]
+                              for path in listing
+                              if path.startswith('MANIFEST.') and
+                              path != 'MANIFEST.BSD-intel']
```

Comment:
MANIFEST.BSD3-intel

Reply:
also look into why this still worked

Reply:
Took a while to get to the bottom of this one, but the code is now fixed.